cmd/minikube/cmd/start.go

```diff
@@ -820,8 +820,14 @@ func validateKubernetesVersions(old *cfg.Config) (string, bool) {
 
 	if nvs.LT(ovs) {
 		nv = version.VersionPrefix + ovs.String()
-		out.ErrT(out.Conflict, "Kubernetes downgrade is not supported, will continue to use {{.version}}", out.V{"version": nv})
-		return nv, isUpgrade
+		exit.WithCodeT(exit.Config, `Error: You have selected version {{.new}}, but your existing "minikube" cluster is
+running version {{.old}}. Non-destructive downgrades are not supported, but you
+can proceed by selecting one of these options:
+
+* Run minikube delete -p minikube to delete the existing cluster
+* Run minikube start -p <new name> to start a new cluster with a different name
+* Run minikube start --kubernetes-version=v1.15.2 or newer to continue using this cluster.`, out.V{"new": nvs, "old": ovs})
```

Comment:
Thanks for doing this! Instead of `v1.15.2`, this should be `{{.new}}`.

Reply:
sorry had missed that, now fixed